corehq/apps/api/tests/test_messaging_event_api.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+from corehq.apps.api.tests.utils import APIResourceTest
+from corehq.apps.api.resources.v0_5 import (
+    MessagingEventResource
+)
+from corehq.apps.sms.tests.data_generator import create_fake_sms
+
+
+class TestMessagingEventResource(APIResourceTest):
+    resource = MessagingEventResource
+    api_name = 'v0.5'
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def _create_sms_messages(self, count, randomize, domain=None):
+        domain = domain or self.domain.name
+        for i in range(count):
+            create_fake_sms(domain, randomize=randomize)
+
+    def _serialized_messaging_event(self):
+        return {
+            "additional_error_text": None,
+            "app_id": None,
+            "content_type": "SMS",
+            "date": "2016-01-01T12:00:00",
+            "domain": "qwerty",
+            "error_code": None,
+            "form_name": None,
+            "form_unique_id": None,
+            # "id": 1,  # ids are explicitly removed from comparison
+            "recipient_id": None,
+            "recipient_type": None,
+            "resource_uri": "",
+            "source": "OTH",
+            "source_id": None,
+            "status": "CMP"
```

Comment:
Is the meaning of these documented anywhere? Or is the plan to convert them to something more meaningful?

Reply:
It's all these things: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/sms/models.py#L886-L1054

Would likely figure out a way to convert them prior to "releasing" the API. initial goal is just to confirm whether this solves the client's problem.